CONTRIBUTORS.rst

```diff
@@ -5,4 +5,4 @@
 
 - `معین باباپور <https://github.com/smoeinbbp>`_
 
-.. - `Your Name <https://github.com/YourGithubUsername>`_
```

Comment:
Please restore this line, keep it as the last line, and then move your name to line 7.